assets/js/app.js

```diff
@@ -21,8 +21,11 @@ import "phoenix_html";
 import { Socket } from "phoenix";
 import { LiveSocket } from "phoenix_live_view";
 import topbar from "../vendor/topbar";
+import LocalTime from "./date";
```

Comment:
What is this used or? I am not a big fan on any JavaScript which is not essentiell.

Reply:
The hook just formats a datetime field. I wanted to display a time for when the story was created. But later found out we don't have timestamps for the story resource. I can go ahead and add timestamps or remove this hook entirely.

Reply:
With phoenix, I haven't found an easier way to get the user's timezone. Most implementations rely on adding the timezone from the browser to the user's session and then using it in elixir land. Either way some javascript is required. A javascript hook for me is much easier as it handles all the formatting and timezone stuff.

Reply:
> I can go ahead and add timestamps or remove this hook entirely.

Please do add the timestamps but also remove this hook and don't display the timestamps at all. Maybe in the future we can use them but right now we don't need that information in the GUI.

Reply:
> With phoenix, I haven't found an easier way to get the user's timezone. 

Yes, that is a tricky one. But I am very sensitive with adding any JavaScript on the client side. I have had so much trouble with that in the past that I avoid it where possible.

Reply:
This is now done